src/movies.js

```diff
@@ -1,16 +1,79 @@
 // Iteration 1: All directors? - Get the array of all directors.
+function getAllDirectors(array){
+    let directorsArr = array.map((elem)=>{
+    return elem.director
```

Comment:
check indentation, remember to add elements inside a function one tab to the right